charts/kyverno-crds/templates/crds.yaml

```diff
@@ -1829,6 +1829,15 @@ spec:
                         type: string
                     type: object
                   type: array
+                result:
```

Comment:
do we need any changes in the crds??

Reply:
I'm also confused with that because the steps you mention 

>change the ConfigMap name here
use command make `kustomize-crd` to get the changes in required files

I did both the step but after that many things get updated in this 3 files
- https://github.com/kyverno/kyverno/blob/main/charts/kyverno-crds/templates/crds.yaml
- https://github.com/kyverno/kyverno/blob/main/definitions/install.yaml
- https://github.com/kyverno/kyverno/blob/main/definitions/install_debug.yaml

which I was not able to understand and after that I had a merge conflict. I tried to solve it but I think it was not done correctly and the merge conflict is still there.

PS: I think closing this PR and opening a new PR will be a good option as I have already messed it up. what do you say?

Reply:
Take a pull from latest pull branch, do the changes and raise a fresh PR.